aio_pika/exchange.py

```diff
@@ -19,13 +20,15 @@ class ExchangeType(Enum):
     HEADERS = 'headers'
     X_DELAYED_MESSAGE = 'x-delayed-message'
     X_CONSISTENT_HASH = 'x-consistent-hash'
+    X_MODULUS_HASH = 'x-modulus-hash'
 
 
 class Exchange:
     """ Exchange abstraction """
 
     def __init__(self, connection, channel: aiormq.Channel, name: str,
-                 type: ExchangeType = ExchangeType.DIRECT, *,
```

Comment:
the [same](https://github.com/mosquito/aio-pika/blob/master/aio_pika/types.py#L8)